language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,944 @@
+{
```

Comment:
<p>After pip installing, there should also be a cell to show how to restart the kernel/runtime, to allow the library to reload and be recognized. Could you copy/paste the relevant restart kernel cells from here: https://github.com/GoogleCloudPlatform/generative-ai/blob/main/language/intro_palm_api.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>thanks - done</p>